planeshifted/planeshifted-guide/phyrexia/race-elf.xml

```diff
@@ -0,0 +1,89 @@
+		<description>
+			<p>As all organic life on New Phyrexia, the Viridian elves are partly inorganic, growing copper from their flesh as a result of the mycosynth that touches everything on the plane. Their organic skin can appear in shades of pale white and bright green.</p>
+			<p class="indent">The elves once resided in the forest of copper trees known as the Tangle. Even before the Phyrexianization of the plane, the Viridian elves were enemies of inorganic and mechanical constructs, frequently coming into conflict with the Myr and the drones created by the Vedalken.</p>
+			<p class="indent">The Viridian elves once looked to the trolls of Tel-Jilad for guidance. The trolls considered the Tel-Jilad (the "Tree of Tales") their sacred home, and they lived within its hollow core. Though the trolls are all but extinct, the elves still hold the Tel-Jilad in reverence. Inscribed into its metal surface is the history of Mirrodin, except near the bottom where all the writings have been etched out.</p>
+			<p class="indent">Due to their long lives, elves will engage in a ceremony known as the Ritual of Rebuking, in which an older elf has their memories erased so that they can start anew and focus on the tasks at hand.</p>
+			<h4>Elf Traits</h4>
+			<p>You have the following racial traits.</p>
+			<p class="indent"><strong><em>Creature Type.</em></strong> You are a Humanoid.</p>
+			<p class="indent"><strong><em>Size.</em></strong> You are Medium (about 5-6 feet tall).</p>
+			<p class="indent"><strong><em>Speed.</em></strong> Your walking speed is 30 feet.</p>
+			<p class="indent"><strong><em>Life Span.</em></strong> 750 years on average.</p>
+			<p class="indent"><strong><em>Copper Spikes.</em></strong> A creature grappling or grappled by you takes piercing damage equal to your proficiency bonus at the start of each of your turns.</p>
+			<p class="indent"><strong><em>Darkvision.</em></strong> You can see in dim light within 60 feet of you as if it were bright light and in darkness as if it were in dim light. You discern colors in that darkness only as shades of gray.</p>
+			<p class="indent"><strong><em>Keen Senses.</em></strong> You have proficiency in the Perception skill.</p>
+			<p class="indent"><strong><em>Iron Scent.</em></strong> As an action, you can pinpoint, by scent, the location of ferrous metal within 30 feet of you.</p>
+			<p class="indent"><strong><em>Mask of the Wild.</em></strong> You can attempt to hide even when you are only lightly obscured by foliage, heavy rain, falling snow, mist, and other natural phenomena.</p>
+			<p class="indent"><strong><em>Siege Monster.</em></strong> You deal double damage to objects and structures. When you damage a Construct creature with an attack or a spell, the attack or spell deals extra damage to the creature equal to your proficiency bonus.</p>
+		</description>
+		<sheet display="false" />
+		<rules>
+			<grant type="Grants" id="ID_INTERNAL_RACIAL_TRAIT_TYPE_HUMANOID" />
```

Comment:
None of the humanoids implemented in the Legacy repo have this grant. I can see it was probably intended to have the elf grant